### PR TITLE
Document is-offline flag

### PR DESCRIPTION
**Pull request by georgecrawford**
In ft-app, we make a best-guess as to whether the User Agent is offline or not, and vary some behaviour accordingly. It's useful to us to document this state along with each event.

### CODE REVIEW

File: docs/event.md [hunk 1]
```diff
@@ -23,11 +23,12 @@ This allows a low barrier to entry for anyone wanting to submit events to Spoor.
...
 		"ip": "123.123.123.123",
 		"layout": "S",											// require('o-grid').getCurrentLayout
-		"orientation": "portrait"								// require('o-viewport').getOrientation
+		"orientation": "portrait",								// require('o-viewport').getOrientation
 		"dimensions": {
 			"width": 1024,
 			"height": 768
-		}
+		},
+		"is-offline": true
```
Comment (triblondon) [inline docs/event.md]: Use of hyphen seems inconsistent with spoor_id, ft_session, user_agent etc.

Comment (georgecrawford) [inline docs/event.md]: Thanks, totally got that wrong!

**Comment by commuterjoy** [comment]
Can this be inferred by the presence of a the time `offset`?

**Comment by georgecrawford** [comment]
I don't think they're quite the same thing, although they will often correlate. `offset` can simply indicate a failed network request, and even if that is because the device is offline, we may not have correctly determined that fact. We try to derive the `is-offline` state, and vary behaviour on that, so it's quite important for us to track it.

**Comment by markstephens** [comment]
I much prefer to have a specific value for things than have many people trying to recalculate something, so it's a :+1: from me.

**Pull request merged**